keyboards/handwired/8key/readme.md

```diff
@@ -0,0 +1,12 @@
+# 8key
+
+A 8-key macropad made by myoung34.
+
+* Keyboard Maintainer: [myoung34](https://github.com/myoung34)
+* Hardware Supported: Pro Micro ATmega32U4, Teensy 2.0
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```